modules/xfem/include/efa/EFAElement2D.h

```diff
@@ -33,6 +33,10 @@ class EFAElement2D : public EFAElement
   std::vector<std::vector<EFAElement2D *>> _edge_neighbors;
   std::vector<EFAFragment2D *> _fragments;
   std::vector<EFAPoint> _local_node_coor;
+  unsigned int _current_cut_plane_idx = 0;
```

Comment:
I know a lot of our code isn't documented like this, but as we add stuff, our convention is to add short doxygen comments for each variable in the header file. I realize this is kind of stating the obvious in this case, but something like this:
```
/// Index of the current cut plane
unsigned int _current_cut_plane_idx = 0;

Reply:
Also, delete the commented out stuff that you didn't end up using both here and below.